cypress_bc98/cypress/e2e/minet/loginRobots.cy.ts

```diff
@@ -0,0 +1,95 @@
+ /// <reference types="cypress" />
+
+import cypress = require("cypress");
+import { LoginPageDepandence, LoginPageRobotEyes, LoginPageRobotHands } from "../../robots/minetApp/LoginPage.cy";
+import { Dependencies,  } from "../../robots/minetApp/Robot";
+import { DashboardPageBaseEyes, DashboardPageDepandence, DashboardPageHands } from "../../robots/minetApp/Dashboard.cy";
+import { AllAssetsEyes, AllAssetsHands } from "../../robots/minetApp/AllAssets.cy";
+import { DetailsPageEyes, DetailsPageHands } from "../../robots/minetApp/DetailsPage.cy";
+import { SellPageEyes, SellPageHands } from "../../robots/minetApp/Sellpage.cy";
+import { BuyPageEyes, BuyPageHands } from "../../robots/minetApp/BuyPage.cy";
+
+const loginEyes= new LoginPageRobotEyes();
+const loginHands=new LoginPageRobotHands();
+const dependencies=new Dependencies();
+const loginPage=new LoginPageDepandence();
+const dashboardPageDepandence=new DashboardPageDepandence();
+const dashboardPageBaseEyes=new DashboardPageBaseEyes();
+const allAssetsEyes=new AllAssetsEyes();
+const allAssetsHands=new AllAssetsHands();
+const detailsPageEyes=new DetailsPageEyes();
+const detailsPageHands=new DetailsPageHands();
+const sellPageEyes=new SellPageEyes();
+const sellPageHands=new SellPageHands();
+const buyPageEyes=new BuyPageEyes();
+const buyPageHands=new BuyPageHands();
+ 
+ describe('minet tesing', () => {  
+    it("test from login page",()=>{
+      loginPage.visitLoginPage()
+      loginEyes.AssertTitle()
+      loginHands.EnterEmail("email","shaikshakeelgr@gmail.com")
+      loginHands.EnterPassword("password","shaik123@")
+      loginHands.ClickOnSignup()
+       cy.wait(2000)
+ 
+      dashboardPageBaseEyes.AssertTitle()
```

Comment:
make sure the actions chainable. check across the code

Reply:
Make the actions chainable like this
 dashboardPageBaseEyes.AssertTitles()
      .AssertButtons()